lib/features/identity_creator/presentation/identity_creator_controller.dart

```diff
@@ -164,7 +167,7 @@ class IdentityCreatorController extends BaseController {
     if (accountId != null) {
       consumeState(_getAllIdentitiesInteractor.execute(
         accountId!,
-        properties: Properties({'email'})
+        properties: Properties({'email', 'sortOrder'})
```

Comment:
What happens if we request this property `sortOrder` with the JMAP server not support default identity?